jobs/database-seeder/templates/pre-start.erb

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+<%
+require 'json'
+require 'shellwords'
+%>
+
+set -o errexit -o nounset
+
+case "<%= p('database-seeder.driver', '') %>" in
```

Comment:
I know I wrote this, but now that I look again I wonder why I didn't just write the whole thing in Ruby instead (since the only inputs are BOSH properties).

Meh, it still works.